mrmustard/physics/gaussian.py

```diff
@@ -421,9 +421,7 @@ def CPTP(
         Y = math.single_mode_to_multimode_mat(Y, len(transf_modes))
     if d is not None and d.shape[-1] == 2:
         d = math.single_mode_to_multimode_vec(d, len(transf_modes))
-    indices = [
-        state_modes.index(i) for i in transf_modes
-    ]  # TODO: do this when calling the method instead of here?
```

Comment:
does this TODO not need further attention?

Reply:
Nope, it's about the old MM and I wanted codefactor to see one fewer TODO..